ilastik/applets/objectExtraction/objectExtractionGui.py

```diff
@@ -407,6 +426,17 @@ def _setDefaults(self):
             plugin.setCheckState(0, Qt.Checked)
             FeatureSelectionDialog.recursiveCheckChildren(plugin, Qt.Checked, exclude_location=True)
 
+            # Update parent states after modifying children
+            self.updateParentState(plugin)
+
+            # Update counter and tooltip
+            checked_count = 0
```

Comment:
What's the point of this count? As far as I can tell, it's introduced in this PR but then not used anywhere.